zaas-service/src/main/resources/application.yml

```diff
@@ -157,12 +157,6 @@ eureka:
         secureHealthCheckUrl: ${apiml.service.scheme}://${apiml.service.hostname}:${apiml.service.port}/application/health
         metadata-map:
             apiml:
-                catalog:
```

Comment:
Could you remove it also from the `zaas-service/src/test/resources/application.yml`?.

Reply:
Sure!